util/src/main/java/com/farao_community/farao/util/SystematicSensitivityAnalysisService.java

```diff
@@ -103,7 +103,7 @@ private static void buildFlowFromNetwork(Network network, Crac crac, Map<Cnec, D
         crac.desynchronize(); // To be sure it is always synchronized with the good network
     }
 
-    private static void applyContingencyInCrac(Network network, ComputationManager computationManager, Contingency contingency) {
+    public static void applyContingencyInCrac(Network network, ComputationManager computationManager, Contingency contingency) {
```

Comment:
Actually I think you can add an apply(Network) method to Contingency interface and implement this method directly in ComplexContingency implementation it will make things easier

Reply:
OK